src/components/Dropdown.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <div class="aggregator__dropdown">
+  <div class="dropdown">
+    <div class="display">
+      {{ displayValue }}
+      <img src="../assets/carretDown.svg">
+    </div>
     <b-form-select
```

Comment:
please move b-form-select inside the .display div and apply the css I suggested below. This way the element will better resize (no fixed width). It looks closer to the design.